docs/Contributing/API-for-contributors.md

```diff
@@ -650,12 +656,10 @@ Content-Type: application/octet-stream
 --------------------------f02md47480und42y
 ```
 
-
 ##### Default response
```

Comment:
@noahtalerman The response should include the `id` of the newly created ABM token (maybe along with renew date, org name and admin? i.e. the same fields (minus the teams it is associated with since there are no associations at that point) shown in the [List ABM tokens page](https://www.figma.com/design/j2M1heOh8eZD6LcUJks6HE/%239956-Add-multiple-Apple-Business-Manager-and-Volume-Purchasing-Program-connections?node-id=5316-2520&t=aiZzwiQ9TagC9xbC-0)?

Reply:
Hey @mna totally makes sense. Please feel free to add that as a commit on top of this PR :)

Reply:
Done.